keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -296,25 +296,25 @@ metadata:
   name: 
 provisioner: [1]
 supportedProtocols: [2]
-accessMode: {"ro", "rw"} [3]
+accessMode: {"ro", "wo", "rw"} [3]
```

Comment:
For the paragraph above, under ### BucketClass:
"...The driver name is used to filter BucketContent events. In dynamic brownfield workflows, the BucketClass contains a reference to a `objectBucketName` which names the existing object store bucket. In static brownfield workflows, the provisioner field is empty, the secret used to grant access to the bucket must be specified, and the `objectBucjketName` is unnecessary. In static brownfield, the secret must be created exactly as a driver would, including endpoint, credentials, bucket name, etc."